lib/rb_shift/client.rb

```diff
@@ -29,7 +29,6 @@ def initialize(url, bearer_token = nil, username = nil, password = nil)
                                              headers:    { Authorization: "Bearer #{bearer_token}" }
     end
 
-    # rubocop:disable Metrics/AbcSize
```

Comment:
@jcechace This comment was marked by rubocop as unnecessary. This commit fixes it, but I'm not sure if this error does not appear on you version of rubocop. If this comment should stay, I will discard this commit, just let me know before merge.